mantidproject/fitbenchmarking
# Added the option to select where the results are saved

#### Description of Work

Fixes #2 
The results are now saved in a "results" folder in /fitbenchmarking/fitbenchmarking/ if no path for another directory is provided. Otherwise, the results are saved in the specified directory. The user can specify a directory by modifying `results_dir` in `example_runScripts.py`, instructions are provided in the comments.

#### Testing Instructions

1. In `example_runScript.py`, set `run_data = "nist"`.
2. Run `example_runScript.py` in the mantidpython console.
3. If you specified a path, there should now be a folder at that path containing the results. Otherwise, the results are in `/fitbenchmarking/fitbenchmarking/results`.
4. Repeat the above process with `run_data = "neutron"`

**Note: ** It is a known issue that `results_output.py` functions cannot be run with group_name `neutron` as it produces a crash, see #22 . Hence, the figures will be saved to the right folder, but for the tables a folder called nist_lower will be created.
